samples/samples.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
rename to `quickstart.js` to make it match other packages, and also to make `repo-tools` happy, should we ever use it for this project?

Reply:
Renamed to quickstart :) repo-tools is on it's way out and will be replaced with synth, so lets leave it out for now :)